setup.py

```diff
@@ -4,6 +4,9 @@
     raise RuntimeError("setuptools >= 41 required to build")
 
 setup(
-    use_scm_version={"write_to": "src/virtualenv/version.py", "write_to_template": '__version__ = "{version}"'},
+    use_scm_version={
+        "write_to": "src/virtualenv/version.py",
+        "write_to_template": 'from __future__ import  unicode_literals;\n\n__version__ = "{version}"',
```

Comment:
did you mean
```suggestion
        "write_to_template": 'from __future__ import  unicode_literals;\n\n__version__ = u"{version}"',
```

Reply:
Not needed, we import unicode literals just before.

Reply:
Ah, wasn't reading it right. FWIW using the prefix is shorter than that future import...

Reply:
I've used the import because that's what we use everywhere else, so consistency.